### ISSUE TITLE
Add a support section to the profile tab

### ISSUE THREAD

**Issue by maartenfv** [teamforus/me#181]
Currently the app has an ¨App setting¨ section and an ¨About¨ section.

We should add an ¨Support¨ section which state the following:

Support@forus.io
Github (which should link to: https://github.com/teamforus/me/issues) 

What do you guys think?

**Comment by maxvisser** [comment]
Gerben is the one answering support. So adding that is his decision. Adding our github is only interesting for the more tech savy people.

**Comment by GerbenBosschieter** [comment]
I've already discussed this Maarten, I agree with this approach.
However I would like to add something to this. Instead of using the support@forus.io email address create a new one, like me@forus.io or feedback@forus.io.  I assume that most people who have the need to contact us while using Me appp will do this because they want send feedback or suggestions for improvement. Not really technical or support questions about how the system works. I like a separation between these difference in how/why people contact us.

**Comment by GerbenBosschieter** [comment]
I agree about Max about referencing to our GitHub, Most of the Me app users will be providers and some regular users. Don't see the need for displaying our Github page.

**Comment by maxvisser** [comment]
I like feedback@forus.io as it tells the user its meant for feedback and not support. :wink:
